langchain/utilities/bibtex.py

```diff
@@ -0,0 +1,182 @@
+"""Util that calls bibtexparser."""
+import logging
+import re
+from pathlib import Path
+from typing import Any, Dict, Iterator, List, Mapping, Optional, Sequence, Type
+
+from pydantic import BaseModel, Extra, root_validator
+
+from langchain.schema import Document
+
+logger = logging.getLogger(__name__)
+
+
+class BibtexparserWrapper(BaseModel):
```

Comment:
@wavefrontshaping why do we have a separate wrapper instead of using the `BaseLoader` interface?